scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: ASSEMBLY
+[id="cnf-topology-aware-lifecycle-operator"]
+= Topology Aware Lifecycle Operator
+include::modules/common-attributes.adoc[]
+:context: cnf-topology-aware-lifecycle-operator
+
+toc::[]
+
+You can use the Topology Aware Lifecycle Operator to manage the software lifecycle of multiple single node Openshift (SNO) clusters. The Operator uses {rh-rhacm-first} policies to perform changes on the target clusters.
+
+:FeatureName: Topology Aware Lifecycle Operator
+include::snippets/technology-preview.adoc[leveloffset=+0]
+
+include::modules/cnf-about-topology-aware-lifecycle-operator-config.adoc[leveloffset=+1]
+
+include::modules/cnf-topology-aware-lifecycle-operator-installation-web-console.adoc[leveloffset=+1]
+
+include::modules/cnf-topology-aware-lifecycle-operator-installation-cli.adoc[leveloffset=+1]
+
+include::modules/cnf-topology-aware-lifecycle-operator-create-policies.adoc[leveloffset=+1]
+
+include::modules/cnf-topology-aware-lifecycle-operator-precache-feature.adoc[leveloffset=+1]
+
+include::modules/cnf-topology-aware-lifecycle-operator-apply-policies.adoc[leveloffset=+1]
+
+[id="additional-resources-topology-aware-lifecycle-operator-installation"]
+== Additional resources
+
+The Topology Aware Lifecycle Operator is supported in a restricted network environment. For more information, see xref:../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks].
+
+The Topology Aware Lifecycle Operator uses `PolicyGenTemplate` CRs. For more information, see xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#ztp-the-policygentemplate_ztp-deploying-disconnected[The PolicyGenTemplate].
```

Comment:
I like having this link in here but I think the statement is too definitive. Policies to be remediated by TALO can be created using PolicyGenTemplate CRs....